middleware.ts

```diff
@@ -1,17 +0,0 @@
-import { NextResponse } from 'next/server'
-import type { NextRequest } from 'next/server'
-
-export const config = {
-  // by only matching on the route where it's needed,
-  // does not interfere with /xrpc (in particular websockets)
-  matcher: '/',
-}
-
-export function middleware(request: NextRequest) {
-  const url = request.nextUrl.clone()
-  if (url.pathname === '/') {
-    url.pathname = '/reports'
-    url.searchParams.set('resolved', 'false')
-    return NextResponse.redirect(url)
-  }
-}
```

Comment:
Why do we no longer need this middleware?

Reply:
Ah, found the client-side redirect 👍

Reply:
The redirect uri is `/`, which is conflicting with the logic here.

Also, the oauth flow uses the `state` param to "remember" the page that was being visited (in case credentials are no longer valid). This mechanism also conflicts with this middleware.